frontend/src/index.tsx

```diff
@@ -51,6 +51,8 @@ import { BrowserRouter, Route, Routes, useLocation } from 'react-router-dom'
 import { QueryParamProvider } from 'use-query-params'
 import { ReactRouter6Adapter } from 'use-query-params/adapters/react-router-6'
 
+document.body.className = 'highlight-light-theme'
```

Comment:
We don't currently support arbitrary html customizations (we currently generate the html from scratch using the config and dependency graph in order to save a network roundtrip by not having to fetch a template on every deploy). This is a workaround for not being able to just set a class attribute on body. Under Vite this should basically be a noop.